test/integration/smoke/test_list_volumes.py

```diff
@@ -140,6 +140,26 @@ def setUpClass(cls):
     def tearDownClass(cls):
         super(TestListVolumes, cls).tearDownClass()
 
+    @classmethod
+    def wait_for_volume_cleanup(cls):
+        """Clean up volumes that were left by previous tests
```

Comment:
```suggestion
        """Wait for volumes to clean up that were left by previous tests
```

this doesn't cleanup, waiting for earlier volumes to tear down.